### DISCUSSION TITLE
赏不了月了

### DISCUSSION THREAD

**Discussion by sb35386057** [share121/inter-knot#5115] [灌水]
大湾区下大雨,本来还想看一下月亮长啥样,这下不用想了,吃月饼就是了
![68747470733a2f2f696d672d7374617469632e6d69686f796f2e636f6d2f636f6d6d756e6974797765622f75706c6f61642f31326138306138663564613339343962303836633439376238636334623564332e706e67](https://github.com/user-attachments/assets/efa4251e-6d67-41bc-9aaf-6986459a1bfa)
![IMG_20240917_220001](https://github.com/user-attachments/assets/18f07876-ef0f-4d19-83f0-dd004f07cf6b)
![IMG_20240917_220012](https://github.com/user-attachments/assets/f4091d88-8887-42cf-8462-a02796f2e247)
希望大家赏月的时候能看得见月亮

**Comment by Fluorine-z** [comment]
![IMG_20240917_190433](https://github.com/user-attachments/assets/d808e10d-ec21-40b0-9687-b4ccce81fc0a)
给你看看我的

**Reply by KELERSAMA** [reply]
今晚月亮真美 看看月亮吧
